scripts/maintenance/git-fame.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+# git-fame allows printing "loc" (lines of code), "coms" (number of commits) and "fils" (number of files)
+# per contributor (using .mailmap)
+#
+# Suggested usage of git fame:
+#-w, --ignore-whitespace  Ignore whitespace when comparing the parent's
+#                           version and the child's to find where the lines
+#                           came from [default: False].
+#  -M  Detect intra-file line moves and copies [default: False].
+#  -C  Detect inter-file line moves and copies [default: False].
+#
+# These slow it down dramatically, but a lot of this has happened in STIR
```

Comment:
```suggestion
#  -w, --ignore-whitespace  Ignore whitespace when comparing the parent's
#                           version and the child's to find where the lines
#                           came from [default: False].
#  -M  Detect intra-file line moves and copies [default: False].
#  -C  Detect inter-file line moves and copies [default: False].
#
# In particular, `-C` takes ~30x longer to run, but a lot of this has happened in STIR
```

Reply:
fyi, leaving out `-C` I get this:

Processing: 100%|█████████████| 1475/1475 [01:14<00:00, 19.70file/s]
```json
{"creators": [
    {"name": "Kris Thielemans"},
    {"name": "Jannis Fischer"},
    {"name": "Sanida Mustafovic"},
    {"name": "Nikos Efthimiou"},
    {"name": "Richard Brown"},
    {"name": "Charalampos Tsoumpas"},
    {"name": "Palak Wadhwa"},
    {"name": "Daniel Deidda"},
    {"name": "Tim Borgeaud"},
    {"name": "Tobias Beisel"},
    {"name": "Matthew Jacobson"},
    {"name": "Alexey Zverovich"},
    {"name": "Ander Biguri"},
    {"name": "Robert Twyman"},
    {"name": "Ludovica Brusaferri"},
    {"name": "Ashley Gillman"},
    {"name": "Ottavia Bertolli"},
    {"name": "Edoardo Pasca"},
    {"name": "Pablo Aguiar"},
    {"name": "Benjamin Thomas"},
    {"name": "Nacer Kerrouche"},
    {"name": "Nikos Dikaios"},
    {"name": "C. Ross Schmidtlein"},
    {"name": "Matthias Ehrhardt"},
    {"name": "Evgueni Ovtchinnikov"},
    {"name": "Patrick Valente"},
    {"name": "David V?lgyes"},
    {"name": "Claire Labbe"},
    {"name": "Katie Dinelle"},
    {"name": "Elise Emond"},
    {"name": "Alexander C. Whitehead"},
    {"name": "Parisa Khateri"},
    {"name": "Alaleh Rashidnasab"},
    {"name": "Yu-jung Tsai"},
    {"name": "Casper da Costa-Luis"},
    {"name": "Rebecca Gillen"},
    {"name": "Holger Kohr"},
    {"name": "tokkot"},
    {"name": "Darren Hague"},
    {"name": "Mustapha Sadki"},
    {"name": "Damiano Belluzzo"}]}
```

Reply:
with all the exclusions? Weird.

I'm running the full thing now but it's taking a while. Dinner first!

Reply:
getting this now
```
  {"name": "Kris Thielemans"},
    {"name": "Jannis Fischer"},
    {"name": "Sanida Mustafovic"},
    {"name": "Nikos Efthimiou"},
    {"name": "Richard Brown"},
    {"name": "Charalampos Tsoumpas"},
    {"name": "Tim Borgeaud"},
    {"name": "Palak Wadhwa"},
    {"name": "Tobias Beisel"},
    {"name": "Daniel Deidda"},
    {"name": "Matthew Jacobson"},
    {"name": "Alexey Zverovich"},
    {"name": "Ander Biguri"},
    {"name": "Ashley Gillman"},
    {"name": "Pablo Aguiar"},
    {"name": "Ludovica Brusaferri"},
    {"name": "Edoardo Pasca"},
    {"name": "Ottavia Bertolli"},
    {"name": "Robert Twyman"},
    {"name": "Nikos Dikaios"},
    {"name": "Nacer Kerrouche"},
    {"name": "Benjamin Thomas"},
    {"name": "Evgueni Ovtchinnikov"},
    {"name": "C. Ross Schmidtlein"},
    {"name": "Matthias Ehrhardt"},
    {"name": "Patrick Valente"},
    {"name": "David Völgyes"},
    {"name": "Claire Labbe"},
    {"name": "Katie Dinelle"},
    {"name": "Elise Emond"},
    {"name": "Alaleh Rashidnasab"},
    {"name": "Alexander C. Whitehead"},
    {"name": "Parisa Khateri"},
    {"name": "Casper da Costa-Luis"},
    {"name": "Rebecca Gillen"},
    {"name": "Yu-jung Tsai"},
    {"name": "Darren Hague"},
    {"name": "Holger Kohr"},
    {"name": "tokkot"},
    {"name": "Damiano Belluzzo"},
    {"name": "Mustapha Sadki"}]}
```
by using the zenodo output (great!), I can't see anymore why it ranks Jannis so high... will have a look

Reply:
My guess is that examples/SAFIR-listmode-virtual-scanner/crystal_map_front.txt is still included in gives me roughly 16k lines of code.

Reply:
Ah yes, didn't realise it's a txt file